moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -321,28 +321,28 @@ bool JogCalcs::cartesianJogCalcs(geometry_msgs::TwistStamped& cmd, JogArmShared&
   Eigen::VectorXd delta_x = scaleCartesianCommand(cmd);
 
   // Convert from cartesian commands to joint commands
-  jacobian_ = kinematic_state_->getJacobian(joint_model_group_);
+  Eigen::MatrixXd jacobian = kinematic_state_->getJacobian(joint_model_group_);
```

Comment:
If you are worried about performance, you could make this static as well

Reply:
done

Reply:
Strangely, making this static causes a crash. Probably due to some obscure Eigen memory management issue?

Reply:
I think the problem is:

> Static variables when used inside function are initialized only once, and then they hold their value even through function calls.

https://www.studytonight.com/cpp/static-keyword.php

Reply:
Sounds like some janky Eigen stuff for sure